include/iscsi-private.h

```diff
@@ -40,6 +40,9 @@ extern "C" {
 #ifndef ARRAY_SIZE
 #define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
 #endif
+#ifndef DIV_ROUND_UP
+#define DIV_ROUND_UP(x, y) (((x) + ((y) - 1)) / (y))
```

Comment:
There are more parentheses in this expression than necessary. I think that ((y) - 1) can be changed into (y) - 1 without changing the result of this expression.

Reply:
done